utils/supabase/client.ts

```
@@ -0,0 +1,14 @@
+import { createBrowserClient } from "@supabase/ssr"
+import type { Database } from "@/utils/supabase/database.types"
+
+let supabaseClient: ReturnType<typeof createBrowserClient<Database>> | null = null
+
+export const getSupabaseClient = () => {
+  if (!supabaseClient) {
+    supabaseClient = createBrowserClient<Database>(
+      process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)